src/prevector.h

```diff
@@ -220,7 +220,10 @@ class prevector {
         }
     }
 
-    prevector() : _size(0) {}
+    prevector() : _size(0) {
+        // Explicitly initialize indirect ptr to avoid "used uninitialized" warning
+        _union.indirect = nullptr;
```

Comment:
I'm super not convinced this is an issue. If we're really concerned, easiest fix is probably require C11 and turn prevector into C++ wrapper around C. More realistically, gcc explicitly supports this, and I believe many other compilers as well.